eq-author/src/App/publish/PublishPage.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Why is the Publish Page here as if it's a new feature in this PR? It was merged in the other day.